README.md

```diff
@@ -11,3 +11,18 @@ npm i @ludex-labs/ludex-sdk-js
 ## Usage
 
 For usage and documentation head over to our [official documentation](https://docs.ludex.gg/ludex-sdks/javascript-sdk).
+
+
+## Pushing new version 
+
+```shell
+npm version <new_version_number>
+```
+
+```shell
+git tag -a <new_version_number> -m "version <new_version_number>"
```

Comment:
don't need this step, the version tag needs to match this format to trigger the workflow: `v1.1.1`